### ISSUE TITLE
Global search operators

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1284]
Hi, can we extend function of Global search ? or normal search like here http://goo.gl/KNOylu

![2016-04-25_2000](https://cloud.githubusercontent.com/assets/5773777/14793688/c6093f88-0b20-11e6-8c23-d94490210a48.png)

now the system works with this logic --> "something%" by default in system search and this is problem if we want extact match the results for example like last 3 (or more) numbers of phone/mobile/fax

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Global search operators](https://trello.com/c/JccjAbGe/104-global-search-operators)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
